src/sentry/south_migrations/0489_content_types.py

```diff
@@ -0,0 +1,1416 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # This was removed in Django 1.8, but we didn't run the migration since
+        # we're still using south
+        # https://github.com/django/django/blob/c686dd8e6bb3817bcf04b8f13c025b4d3c3dc6dc/django/contrib/contenttypes/migrations/0002_remove_content_type_name.py
+        db.commit_transaction()
+        db.execute("ALTER TABLE django_content_type DROP COLUMN name;")
```

Comment:
If we drop this column now, will we run into problems when we switch to django migrations later on?

Reply:
Might be a problem. I actually have an issue with the client build failing related to this migration, so I might just hold off here until we move over to Django migrations.

Reply:
:+1: